test/utils.js

```diff
@@ -37,70 +39,68 @@ function compareWarnings ( actual, expected ) {
 			}
 
 			return clone;
-		}),
+		} ),
 		expected.map( warning => {
 			if ( warning.frame ) {
 				warning.frame = deindent( warning.frame );
 			}
 			return warning;
-		})
+		} )
 	);
 }
 
-function deindent(str) {
-	return str.slice(1).replace(/^\t+/gm, '').replace(/\s+$/gm, '').trim();
+function deindent ( str ) {
+	return str.slice( 1 ).replace( /^\t+/gm, '' ).replace( /\s+$/gm, '' ).trim();
 }
 
-function executeBundle(bundle) {
+function executeBundle ( bundle ) {
 	return bundle
-		.generate({
+		.generate( {
 			format: 'cjs'
-		})
-		.then(cjs => {
-			const m = new Function('module', 'exports', cjs.code);
+		} )
+		.then( cjs => {
+			const m = new Function( 'module', 'exports', cjs.code );
 
 			const module = { exports: {} };
-			m(module, module.exports);
+			m( module, module.exports );
 
 			return module.exports;
-		});
+		} );
 }
 
-function extend(target) {
-	[].slice.call(arguments, 1).forEach(source => {
+function extend ( target ) {
+	[].slice.call( arguments, 1 ).forEach( source => {
 		source &&
-			Object.keys(source).forEach(key => {
-				target[key] = source[key];
-			});
-	});
+		Object.keys( source ).forEach( key => {
+			target[ key ] = source[ key ];
+		} );
+	} );
 
 	return target;
 }
 
-function loadConfig(path) {
+function loadConfig ( configFile ) {
 	try {
-		return require(path);
-	} catch (err) {
-		console.error(err.message);
-		console.error(err.stack);
-		throw new Error(
-			`Failed to load ${path}. An old test perhaps? You should probably delete the directory`
-		);
+		return require( configFile );
+	} catch ( err ) {
+		const dir = path.dirname( configFile );
+		console.warn( `Failed to load ${configFile}.\nRemoving directory ${dir} as this is probably the result of a missing test.` );
```

Comment:
A syntax error in a configuration file may be a mistake when editing _config.js and shouldn't result in removing the folder... perhaps we can filter specifically to an ENOENT error?

Reply:
Fixed. Now the directory is only removed for a MODULE_NOT_FOUND error (which is the ENOENT for `require`).